### ISSUE TITLE
Add more keyword

### ISSUE THREAD

**Issue by RioChndr** [RioChndr/jaksel-language#2]
Add more keyword and parser to make it richer

**Comment by fzn0x** [comment]
else if seems like good start

**Comment by fzn0x** [comment]
I thought that we could add a parser to adopt import export using ESM or CJS, but like your quote on social media, we don't need to add anything advanced as this is just for fun.

So my recommendation is only to add basic javascript keywords (like function, btw we can add 'use strict' too to this project regex parser).

**Comment by RioChndr** [comment]
`else if` already added at #9, I love that keyword.

**Comment by RioChndr** [comment]
I agree to create function, need to check other keyword jaksel to fit this usage

**Comment by pulunomoe** [comment]
function = fyp, nama function sama paramnya harus pake hashtag
returnnya jujurly

`#fyp #hitung #satu #dua
`   jujurly satu + dua

**Comment by permanar** [comment]
Lol, this is awesome. 
I really want to contribute to this but didn't really understand and familiar with 'Jaksel' culture so much (since I don't live there).

So yeah, is there something we gotta do? I think if we use the **Projects Board** it would easier to track some of the workload! 
I can help setting it up.

**Comment by RioChndr** [comment]
@permanar I already edit this issue description. If you want to contribute, feel free to add your PR

**Comment by ditatompel** [comment]
Can you add async await or promise resolve please?

**Comment by RioChndr** [comment]
I add it on list, @ditatompel

**Comment by xdendix** [comment]
healing, kurs = kurang, kureng = kurang. mungkin bisa dipertimbangkan, bang. haha

**Comment by FathanAkram-app** [comment]
overthinking = async function
makesure = await
hehe bisa kali ya

**Comment by hkar19** [comment]
i propose:
`literally` : `const`
`misalnya` : `let`
